src/components/ProgressForm/ProgressHeader.tsx

```diff
@@ -19,6 +28,25 @@ const ProgressHeader: FC<progressHeaderProps> = ({
                 <p className={styles.bannerPara}>
                     Let&apos;s try to avoid having zero days
                 </p>
+                {updateType === 'Standup' ? (
```

Comment:
Please use a constant for this 'standup'

Reply:
done